productPage.php

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./css/styles.css">
+    <title>Product Page</title> <!-- Name will be changed from product name -->
+</head>
+
+<body>
+    <!-- Header wird von Kevin Junker implementiert (php import) -->
+    <header>
+        <nav>
+
+        </nav>
+        <img src="" alt="">
+        <nav>
+
+        </nav>
+    </header>
+    <main id="product-seite">
+        <div class="container">
+            <div class="row">
+                <div class="col-2">
+                    <img id="produkt-bild" src="./img/testBild.png" alt=""> <!-- Src wird dynamisch nach Produkt ausgewählt-->
+                </div>
+                <div class="col-2" id="produkt-eigenschaften">
+                    <p style="font-weight: bold">Produkteigenschaften</p>
+                    <table id="eigenschafts-tabelle">
+                        <!-- Eigenschaften werden dynamisch hinzugefügt -->
+                        <tr class="eigenschaft-row">
+                            <td>Eigenschaft 1</td>
+                            <td>Wert 1</td>
+                        </tr>
+                        <tr class="eigenschaft-row">
+                            <td>Eigenschaft 2</td>
+                            <td>Wert 2</td>
+                        </tr>
+                        <tr class="eigenschaft-row">
+                            <td>Eigenschaft 3</td>
+                            <td>Wert 3</td>
+                        </tr>
+                    </table>
+                </div>
+                <div class="col-1-5" style="float:right">
+                    <div id="bestell-section">
+                        <div id="preis-section">
+                            <p>Preiskarte</p>
+                            <!-- Preis wird dynamisch hinzugefügt -->
+                            <p>UVP: <strong>20.99€</strong></p>
+                            <p>Unser Preis: <strong>14.99€</strong></p>
```

Comment:
Preise sollten in einer Reihe stehen und nach dem Mockup lautend, anders benannt werden (alter Preis und aktueller Preis) (Alter Preis für eventuelle Rabattaktionen)